tests/Cache/CacheClientTest.php

```diff
@@ -3784,6 +3785,46 @@ public function testSortedSetLengthByScoreWithMinScoreLargerThanMaxScore_Returns
         $this->assertEquals(MomentoErrorCode::INVALID_ARGUMENT_ERROR, $response->asError()->errorCode());
     }
 
+    public function testSortedSetUnionStore_HappyPath()
+    {
+        $sortedSetName1 = uniqid();
+        $sortedSetName2 = uniqid();
+        $sortedSetName3 = uniqid();
+
+        $elements1 = [
+            "foo" => 1.0,
+            "bar" => 2.0,
+            "baz" => 3.0,
+        ];
+
+        $elements2 = [
+            "abc" => 4.0,
+            "def" => 5.0,
+            "hij" => 6.0,
+        ];
+
+        $this->client->sortedSetPutElements($this->TEST_CACHE_NAME, $sortedSetName1, $elements1);
+        $this->client->sortedSetPutElements($this->TEST_CACHE_NAME, $sortedSetName2, $elements2);
+        $response = $this->client->sortedSetUnionStore(
+            $this->TEST_CACHE_NAME,
+            $sortedSetName3,
+            [
+                ["setName"=>$sortedSetName1, "weight"=>1],
+                ["setName"=>$sortedSetName2, "weight"=>1],
+            ],
+            SortedSetUnionStoreAggregateFunction::SUM,
+            60
+        );
+        $this->assertNull($response->asError());
+        $this->assertNotNull($response->asSuccess(), "Expected a success but got: $response");
+        $this->assertEquals(6, $response->asSuccess()->length());
+
+        $this->client->sortedSetFetchByScore($this->TEST_CACHE_NAME, $sortedSetName3);
+        $this->assertNull($response->asError());
+        $this->assertNotNull($response->asSuccess(), "Expected a hit but got: $response");
+        $this->assertEquals(6, $response->asSuccess()->length());
```

Comment:
Can you also verify the scores are as expected, add a test for the other two aggregation variants, and one for different weightings as a sanity check?

Reply:
I am planning on adding a ton of testing as a followup but want to go ahead and get this approved. I still need to cut a release to use in the `momento-php-redis-client`.